### PR TITLE
docs(): Edited README.md to include regular devs

### PR DESCRIPTION
**Pull request by rickschmoo**
Minor wording change to include developers who want to simply use the library (as well as those who want to contribute)

### CODE REVIEW

**Comment by rickschmoo** [comment]
I just completed Google form for CLA

File: README.md [hunk 1]
```diff
@@ -29,8 +29,7 @@ Salesforce Accessibility Automated Testing Libraries and Tools (@sa11y packages)
 ## Docs
 
 -   [Developer Guidelines](./CONTRIBUTING.md)
-    -   Refer to [Quick start](./CONTRIBUTING.md#quick-start) instructions to get started, if you are a
-        developer looking to Contribute
+    -   If you are a developer looking to use the library or to contribute, refer to [Quick start](./CONTRIBUTING.md#quick-start) instructions to get started
```
Comment (mohanraj-r) [inline README.md:32]: @rickschmoo Thanks for the PR. There are 2 target audiences for this library. The Contributing guidelines is for developers looking to make changes to the source code of the library itself. For users who simply want to install and use the library the contributing guidelines does not apply - they can just install and use the library using npm/yarn tools.

Comment (rickschmoo) [reply README.md]: @mohanraj-r Got it. See now. Any idea why I'm getting the PR title linter error?

Comment (mohanraj-r) [reply README.md]: The PR title should follow the format as described in https://github.com/mohanraj-r/sa11y/blob/update_fix_docs/CONTRIBUTING.md#pull-request-title

**Comment by rickschmoo** [comment]
Doc was already edited

**Pull request closed without merging**